packages/runtime/container-runtime/src/summary/summarizer.ts

```diff
@@ -87,6 +87,7 @@ export class Summarizer extends EventEmitter implements ISummarizer {
 		private readonly runCoordinatorCreateFn: (
 			runtime: IConnectableRuntime,
 		) => Promise<ICancellableSummarizerController>,
+		private readonly listenToDeltaManagerOps: boolean = true,
```

Comment:
let's open a bug into the future to remove listening on delta manager completely. I assume that requires all clients out there to be on 2.x version of loader, so it may take a while till it happens.

Reply:
[AB#3883](https://dev.azure.com/fluidframework/internal/_workitems/edit/3883)